modules/sdk-coin-sol/src/lib/constants.ts

```diff
@@ -114,3 +114,188 @@ export const ataInitInstructionIndexes = {
 
 export const nonceAdvanceInstruction = 'AdvanceNonceAccount';
 export const validInstructionData = '0a00000001000000';
+
+export const SOL_MAINNET_TOKENS = {
```

Comment:
Shouldn't this live in statics instead? 

https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts

Reply:
I was just trying to make a structure that was easy to check based on name and/or mint address if it was supported in recovery, like how its done for say tron tokens. seemed easier than dealing with the `solToken` structure in that file

Reply:
But repeating this large list is not desirable. By adding/using statics, it enables other packages to use this token data in a more generic way without havbing to import individual coin modules.


For example:  UI's can ingest this data from the statics lib instead of having to switch between coin modules to get TRON tokens vs SOL tokens.

Reply:
Also reduces the risk of mis-configed data where statics defines a token one way but this hard coded list has it configured a seperate way.